addons/html_editor/static/tests/link/popover.test.js

```diff
@@ -35,6 +35,25 @@ describe("should open a popover", () => {
         setContent(el, "<p>this []is a <a>link</a></p>");
         await expectElementCount(".o-we-linkpopover", 0);
     });
+    test("should close a popover and remove link when click outside of a link without href (2)", async () => {
```

Comment:
Content of the test does not seem to match the title.

Reply:
Ahh my bad, I didn't adapt the title after I realized I only needed to check the link is padded properly :grimacing: